pkg/auth/cell.go

```diff
@@ -39,46 +38,46 @@ var Cell = cell.Module(
 		// Always fail auth handler provides support for auth type "always-fail" - which always fails.
 		newAlwaysFailAuthHandler,
 	),
-	cell.Config(config{MeshAuthMonitorQueueSize: 1024}),
+	cell.Config(config{MeshAuthQueueSize: 1024}),
 	cell.Config(MTLSConfig{}),
 )
 
 type config struct {
-	MeshAuthMonitorQueueSize int
+	MeshAuthQueueSize int
 }
 
 func (r config) Flags(flags *pflag.FlagSet) {
-	flags.Int("mesh-auth-monitor-queue-size", r.MeshAuthMonitorQueueSize, "Queue size for the auth monitor")
+	flags.Int("mesh-auth-queue-size", r.MeshAuthQueueSize, "Queue size for the auth manager")
```

Comment:
should we add the actual purpose of "authentication" to the flag description? or maybe even mention the signalmap. currently it reads as THE queue of the auth manager. 

(we will have some more channels in the near future (GC, ..).)

Reply:
Maybe we do not need to expose all of them separately to the user? For example, use a single queue-size parameter to control the set up of multiple queues/channels?

I'll leave this be for now, we can fine tune this when these additional queues are introduced.

Reply:
agree :ok_hand: marking as resolved